python-json-rbac/jwt_keyman.py

```
@@ -0,0 +1,291 @@
+
+# Add the package to the path
+sys.path.insert(0, str(Path(__file__).parent))
+
+try:
+    from key_manager import (
+        get_key_manager,
+        validate_current_secret,
+        generate_secure_secret,
+        create_rotation_plan,
+    )
+    from config import get_secret_info, generate_new_secret
```

Ruff (0.12.2)

`config.generate_new_secret` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)